cmd/editorconfig-checker/main.go

```diff
@@ -18,8 +18,8 @@ import (
 // version is dynamically set at compiletime
 var version string
 
-// defaultConfigFilePath determines where the config is located
-const defaultConfigFilePath = ".ecrc"
+// defaultConfigFileNames determines the file names where the config is located
+var defaultConfigFileNames = []string{".ecrc", ".ecrc.json", ".editorconfig-checkerrc", ".editorconfig-checkerrc.json"}
```

Comment:
For the moment, we keep support for `.ecrc`, `.ecrc.json`, but when #367 lands, we will drop support for them.

Reply:
Was .edrc.json really supported ?

Reply:
No, `.ecrc.json` wasn't supported, but as we're adding support for `.editorconfig-checkerrc.json`, it might makes sense to also support `.ecrc.json`. But yeah, maybe, it is better to not support `.ecrc.json`, as we're probably dropping support for it in the next major version.